rocky/reports/views/base.py

```diff
@@ -433,24 +433,16 @@ class ViewReportView(ObservedAtMixin, OrganizationView, TemplateView):
 
     def setup(self, request, *args, **kwargs):
         super().setup(request, *args, **kwargs)
-        self.report_id = request.GET.get("report_id")
-        self.report_ooi = self.octopoes_api_connector.get(
-            Reference.from_str(f"{self.report_id}"), valid_time=self.observed_at
-        )
+        self.report_ooi = self.get_report_ooi(request.GET.get("report_id"))
+        self.report_data, self.input_oois, self.report_types = self.get_report_data()
 
     def get(self, request, *args, **kwargs):
         if "json" in self.request.GET and self.request.GET["json"] == "true":
-            self.bytes_client.login()
-            data = json.loads(self.bytes_client.get_raw(raw_id=self.report_ooi.data_raw_id))
-
             response = {
                 "organization_code": self.organization.code,
                 "organization_name": self.organization.name,
                 "organization_tags": list(self.organization.tags.all()),
-                "data": {
-                    "report_data": {},
-                    "post_processed_data": data,
-                },
+                "data": json.loads(json.dumps(self.report_data)),
```

Comment:
Do we need this dumps and loads? They seem redundant? If the response.data is already dumped as json somewhere.